### DISCUSSION TITLE
Has anyone else experienced the reported voltage jumping up immediately after an OTA update?

### DISCUSSION THREAD

**Discussion by nekorevend** [emporia-vue-local/esphome#113] [General]
Recently I had made some calibration changes to my config and let ESPHome install it onto the Emporia, but after the install the reported voltages immediately jumped from about 119V to about 130V on both phases. I've confirmed with 3 other voltmeters along with the voltage reported by my utility's meter and all of them still claim around the 119-120V range.

My calibration tweaks did not touch the voltages at all, and reverting the config did not fix the issue. This suggests that the hardware itself thinks the voltage is 130V, but only suddenly after an OTA. So maybe it is a hardware issue, or maybe some change in this repo or a change in ESPHome upstream changed how it interpreted the data from the chip.

Hard to diagnose right now. I would like to know if anyone else has experienced this too!

**Comment by flaviut** [comment]
Not sure what's wrong here unfortunately, or how to debug this. I'm assuming you've already tried flipping the breaker to reboot the vue?

There's been no changes to voltage reporting since the initial release.

----

Let me correct myself a bit here, this can be debugged by grabbing [a copy of the schematic](https://github.com/emporia-vue-local/emporia-vue2-reversing/blob/master/vue2_schematic.pdf) and using a multimeter to try and figure out what's wrong. But that involves working with **very very dangerous** mains voltages, and should not be done by anyone not a trained professional.

**Comment by nekorevend** [comment]
Yes I've tried rebooting the Emporia by flipping the breaker. I don't think I'll try to debug this, and have just added a 0.9182148 multiplier to every filter list, and it works ok.

More evidence that it's a hardware issue is that all the wattages are also increased by the same amount. So the incorrect voltage is being used in the watt calculation by the chip itself, which is before ESPHome can do anything about it.
